src/world/format/io/data/BedrockWorldData.php

```diff
@@ -155,18 +154,12 @@ protected function load() : CompoundTag{
 		}
 
 		$version = $worldData->getInt(self::TAG_STORAGE_VERSION, Limits::INT32_MAX);
-		if($version === Limits::INT32_MAX){
-			throw new CorruptedWorldException(sprintf("Missing '%s' tag in level.dat", self::TAG_STORAGE_VERSION));
-		}
 		if($version > self::CURRENT_STORAGE_VERSION){
 			throw new UnsupportedWorldFormatException("LevelDB world format version $version is currently unsupported");
 		}
 		//StorageVersion is rarely updated - instead, the game relies on the NetworkVersion tag, which is synced with
 		//the network protocol version for that version.
 		$protocolVersion = $worldData->getInt(self::TAG_NETWORK_VERSION, Limits::INT32_MAX);
-		if($protocolVersion === Limits::INT32_MAX){
-			throw new CorruptedWorldException(sprintf("Missing '%s' tag in level.dat", self::TAG_NETWORK_VERSION));
-		}
```

Comment:
Why is this changed?

Reply:
I didn't change it, maybe it's a github bug

Reply:
Diff suggests it came from this commit: https://github.com/pmmp/PocketMine-MP/pull/5835/commits/8d7023f9ad7ac9c72b09b55bb09cfce4f33d2ecf

Reply:
Should be fixed now